scss/_alerts.scss

```diff
@@ -1,17 +1,18 @@
 .flash-messages {
-  margin-top: 15px;
-  margin-bottom: 15px;
+  margin-top: 16px;
```

Comment:
I'm unsure why Hound is reporting them in that order—the opposite is true. You can see that in our SCSS lint file in #18. I'll open an issue there.